### PR TITLE
Reasons to use this project vs googleapis/google-api-nodejs-client

### PR DESCRIPTION
**Pull request by Maxim-Mazurok**
While searching for DialogFlow libs, I rediscovered this project: https://github.com/googleapis/google-api-nodejs-client

It has virtually all Google APIs written in TypeScript. But you can't use it on client-side (because it uses `fs`) and it doesn't have [client-side auth lib](https://developers.google.com/identity/sign-in/web/reference) and [client lib](https://github.com/google/google-api-javascript-client/issues/432#issuecomment-435523106)

Please, let me know what made you use this project. Am I missing any reasons?

### CODE REVIEW

**Comment by declanvong** [comment]
Yep, in mine I just said `These definitions are for the Google API Javascript Client: https://github.com/google/google-api-javascript-client` and that was about it. I think your explanation is fine though. 

TLDR is just google-api-nodejs-client != google-api-javascript-client, right?

**Comment by Maxim-Mazurok** [comment]
Thank you guys a lot! I finally wrapped my head around how the things work with these APIs.
Please, check the updated version.

> would it make sense to focus on clarifying that this project is intended to generate type definitions for gapi?

Yes, in this update I tried to make it clear and moved focus to `gapi`.

> TLDR is just google-api-nodejs-client != google-api-javascript-client, right?

Right, I just wasn't really sure what the key differences are.
Basically, client libraries are generated on the fly from discovery service by gapi, which is closed-source.
